### PR TITLE
Lack of ground-truth labels is a major challenge in healthcare data

### PR DESCRIPTION
**Pull request by XieConnect**
For many healthcare tasks, it is often required to have data manually labeled by clinician experts (i.e., expert chart review; such as in phenotyping). The learnt features also need to be validated by clinicians to pass publication standards. This process is costly, and lack of ground-truth labeled medical data is a major obstacle, affecting EHR data, medical images, and some omics data.

### CODE REVIEW

File: sections/03_categorize.md [hunk 1]
```diff
@@ -167,6 +167,35 @@ advances to practical game-changers.
 Additionally, unique barriers exist in this space that may hinder progress in
 this field.
 
+###### Lack of ground-truth labels in data, and high cost in post-study validation by clinician experts.
+
+*Lack of true labels is perhaps among the biggest obstacles for electronic
```
Comment (cgreene) [inline sections/03_categorize.md]: I think you can remove the emphasis markers here since we were using those to mark filler text. Also check for trailing spaces (suspect you can auto-remove them - `atom`'s reflow selection will auto correct) and >80char lines. Particularly the line length can affect the ease of review.

File: sections/03_categorize.md [hunk 2]
```diff
@@ -205,6 +234,12 @@ we see deep learning currently being used" section below.*
 expensive, so it may be helpful. I leave it here as a stub in case someone wants
 to take it on.*
 
+*(Wei Xie; @XieConnect): I personally do not think this subsection is necessary.
```
Comment (cgreene) [inline sections/03_categorize.md]: Sounds good - if nobody else takes it on we should feel free to drop it 👍

File: sections/03_categorize.md [hunk 3]
```diff
@@ -167,6 +167,35 @@ advances to practical game-changers.
 Additionally, unique barriers exist in this space that may hinder progress in
 this field.
 
+###### Lack of ground-truth labels in data, and high cost in post-study validation by clinician experts.
```
Comment (cgreene) [inline sections/03_categorize.md]: Can we call this something like "The challenge of pyrite standards" to contrast with gold standards that many fields have access to?

File: sections/03_categorize.md (hunk 1 above)
Comment (cgreene) [inline sections/03_categorize.md]: I think this many assume too much about the reader. We might want to also point out the difference between what is true and what we think is true.

File: sections/03_categorize.md [hunk 4]
```diff
@@ -167,6 +167,35 @@ advances to practical game-changers.
...
+
+*Lack of true labels is perhaps among the biggest obstacles for electronic 
+health records(EHR)-based learing studies (e.g., phenotyping). Popular deep 
+learning (and machine learning) methods are mostly classification problem, 
+which require ground-truth labels (i.e., response variables) for training. 
+For EHR, unfortunately, this means we need to hire multiple expert clinicians 
+to manually read and annotate individual patients' records to assign "true" 
+labels (i.e., expert chart review). Sometimes, the learnt features also need to 
+be manually validated and interpreted by clinicians. This can be quite costly
```
Comment (cgreene) [inline sections/03_categorize.md]: This ties in directly to #170 which @traversc brought up. Imagine deep learning makes some task very cheap, but since manual analysis is still required, have we gained anything?

File: sections/03_categorize.md [hunk 5]
```diff
@@ -167,6 +167,35 @@ advances to practical game-changers.
 Additionally, unique barriers exist in this space that may hinder progress in
 this field.
 
+###### Lack of ground-truth labels in data, and high cost in post-study validation by clinician experts.
+
+*Lack of true labels is perhaps among the biggest obstacles for electronic 
+health records(EHR)-based learing studies (e.g., phenotyping). Popular deep 
+learning (and machine learning) methods are mostly classification problem, 
+which require ground-truth labels (i.e., response variables) for training. 
+For EHR, unfortunately, this means we need to hire multiple expert clinicians
```
Comment (cgreene) [inline sections/03_categorize.md]: Is the difference in this case from other fields like imagenet the amount of training required for people to provide input? Basically - anybody can classify images but analysis of EHRs requires an expensive physician. I think this is what you are saying. I'd suggest making that point explicitly.

File: sections/03_categorize.md [hunk 6]
```diff
@@ -167,6 +167,35 @@ advances to practical game-changers.
...
+*Lack of true labels is perhaps among the biggest obstacles for electronic 
+health records(EHR)-based learing studies (e.g., phenotyping). Popular deep 
+learning (and machine learning) methods are mostly classification problem, 
+which require ground-truth labels (i.e., response variables) for training. 
+For EHR, unfortunately, this means we need to hire multiple expert clinicians 
+to manually read and annotate individual patients' records to assign "true" 
+labels (i.e., expert chart review). Sometimes, the learnt features also need to 
+be manually validated and interpreted by clinicians. This can be quite costly 
+in time and money [doi: 10.1016/j.ijmedinf.2016.09.014]. Due to resource limitations, 
+many existing works in this direction (including many referenced in this review)
```
Comment (cgreene) [inline sections/03_categorize.md]: Can you reference a few of them here? I think #25 falls into this. Might be nice to at least provide a quick group so that readers can get there.

File: sections/03_categorize.md [hunk 7]
```diff
@@ -167,6 +167,35 @@ advances to practical game-changers.
...
+most sample sizes with expert validations are between 100 to 300 patients, 
+which is quite small even for simple machine learning, not to mentioned data-hungry 
+deep learning models.*
+
+*This issue also common to many related domains -- around medical images, omics. 
+(thus the motivation for semi-supervised learning as briefly mentioned in earlier sections)*
+
+*In summary, this subsection emphasize on two important aspects that are not 
+addressed by other sections: 
+1) Data (and sometimes learnt features/representations)
```
Comment (cgreene) [inline sections/03_categorize.md]: Can this be solved by one large well-annotated clinical resource & clever transfer learning to work in other domains, or do we need large well-annotated resources for every domain? If you were going to design this with a finite amount of funding, where would *you* put the funding? Data, methods, or both?

File: sections/03_categorize.md [hunk 8]
```diff
@@ -167,6 +167,35 @@ advances to practical game-changers.
...
+deep learning models.*
+
+*This issue also common to many related domains -- around medical images, omics. 
+(thus the motivation for semi-supervised learning as briefly mentioned in earlier sections)*
+
+*In summary, this subsection emphasize on two important aspects that are not 
+addressed by other sections: 
+1) Data (and sometimes learnt features/representations) 
+need to be validated by clinicial experts.  
+2) The amount of expert labels needs to be sufficient large.*
```
Comment (cgreene) [inline sections/03_categorize.md]: Can we provide some guidance on what `sufficient` is?

File: sections/03_categorize.md (hunk 4 above)
Comment (cgreene) [inline sections/03_categorize.md]: @traversc - I think you can file a pull request against @XieConnect's repository to mention #170. As soon as that gets approved, I think your contribution would show up here.

**Comment by XieConnect** [comment]
@cgreene Thank you very much for your detailed and encouraging feedback! I'll address your comments first to get the pull request merged. And then I'll continue to expand with concrete examples/detailed writing in a separate pull request.

**Comment by cgreene** [comment]
@XieConnect - I'm looking forward to it! I'm very excited about this section's ability to open a can of worms related to practicality that we really haven't touched on yet. It also raises an important difference between other fields where deep learning has been successful and this field.

**Comment by cgreene** [comment]
@XieConnect - I would love to see you get the revisions for this together so that we can merge it in. @traversc has raised points in a new section (#208) that will relate to the interpretation bit that you have been contributing. We'll certainly keep revising both sections, but since this one brought up the discussion that we're finishing there, having this go in first would be great. Thanks!

File: sections/03_categorize.md (hunk 3 above)
Comment (agitter) [inline sections/03_categorize.md]: Is 'pyrite standard' a common term?  I understand what you mean and it's clever if you coined it, but I had to think about it for a second or two.  We might need to explain the phrase if we use it.  I've also heard this (imperfect 'gold standard') called a 'bronze standard'.

File: sections/03_categorize.md [hunk 9]
```diff
@@ -167,6 +167,35 @@ advances to practical game-changers.
 Additionally, unique barriers exist in this space that may hinder progress in
 this field.
 
+###### Lack of ground-truth labels in data, and high cost in post-study validation by clinician experts.
+
+*Lack of true labels is perhaps among the biggest obstacles for electronic 
+health records(EHR)-based learing studies (e.g., phenotyping). Popular deep 
+learning (and machine learning) methods are mostly classification problem,
```
Comment (agitter) [inline sections/03_categorize.md]: 'classification problems' (plural)

File: sections/03_categorize.md (hunk 7 above)
Comment (agitter) [inline sections/03_categorize.md]: In imaging, we are starting to see papers where groups have made investments in physician time to acquire labeled training data (e.g. #151, #207).  Does that fit here or are we discussing that elsewhere (I haven't read all the related pull requests).

To @cgreene's point, I think this a very interesting topic.  Somewhere (Twitter I guess?) I've seen discussion about data versus methods and how companies may be releasing their deep learning methods as open source code but keeping their valuable datasets private.

File: sections/03_categorize.md (hunk 2 above)
Comment (agitter) [inline sections/03_categorize.md]: Even if it is not a problem, so we want to quickly state what @XieConnect describes regarding why it is not a problem in this domain?  I'm not sure where that would fit if it isn't a sub-section here.  It could be a brief remark in the 'Hardware limitations and scaling' sub-section of Discussion if storage is not a problem in any of the domains we cover.

File: sections/03_categorize.md (hunk 3 above)
Comment (cgreene) [inline sections/03_categorize.md]: I don't think it's a common term. I did coin it, but I'm almost certainly not the first person to do so. I've been using it for a few years for bio gold standards (particularly in gene function prediction), but bronze works for me too :)

Comment (agitter) [inline sections/03_categorize.md]: I don't think it matters what we call these imperfect gold standards so with whatever you prefer.  My main suggestion is that we explain what we mean by "pyrite" or "bronze" standard in the subsequent sentence.

**Comment by cgreene** [comment]
@XieConnect : I wanted to ping you on this. Can you come back to this and make edits to address the comments? I would love to get this merged so that we can bring in @traversc contribution and begin to refine this section.

**Comment by cgreene** [comment]
Ok - I think that I succeeded in this. I had to do a manual merge but it seems to have worked. I'll go back to try to address these comments tomorrow.
